packages/@pollyjs/adapter-xhr/tests/integration/adapter-test.js

```diff
@@ -4,6 +4,7 @@ import adapterTests from '@pollyjs-tests/integration/adapter-tests';
 import adapterBrowserTests from '@pollyjs-tests/integration/adapter-browser-tests';
 import adapterIdentifierTests from '@pollyjs-tests/integration/adapter-identifier-tests';
 import InMemoryPersister from '@pollyjs/persister-in-memory';
+import { Buffer } from 'buffer/';
```

Comment:
curious what the `/` is needed for?  My guess is something to do with the resolver to use the `index` file vs what pkg.js declares as the entry point

Reply:
Nevermind found the answer:
> To require this module explicitly, use require('buffer/') which tells the node.js module lookup algorithm (also used by browserify) to use the npm module named buffer instead of the node.js core module named buffer!